distribution/lib/Standard/Table/0.0.0-dev/src/Data/Conversions/Convertible_To_Rows.enso

```diff
@@ -16,12 +16,20 @@ type Convertible_To_Rows
        - getter: Get the value for a specified row.
     Value length:Integer (getter : Integer->Any)
 
+    ## PRIVATE
+       Return the iterator values as a `Vector`.
+    to_vector : Vector Any
+    to_vector self = 0.up_to self.length . map self.getter
+
 ## PRIVATE
 Convertible_To_Rows.from that:Vector = Convertible_To_Rows.Value that.length that.get
 
 ## PRIVATE
 Convertible_To_Rows.from that:Array = Convertible_To_Rows.Value that.length that.get
 
+## PRIVATE
+Convertible_To_Rows.from that:List = Convertible_To_Rows.Value that.length that.get
```

Comment:
I wonder if we should do:
```suggestion
## PRIVATE
Convertible_To_Rows.from that:List = Convertible_To_Rows.from that.to_vector
```

It will give us the same results, but since we convert to vector we will get O(1) access of `get` and avoid the O(N^2) cost accumulation when traversing the whole structure.